src/EmbeddingSpMDMAvx2.cc

```diff
@@ -167,5 +167,29 @@ template bool EmbeddingSpMDMBlockSize1_(
     float* out,
     bool is_weight_positional);
 
+template bool EmbeddingSpMDMBlockSize1_(
```

Comment:
This is a duplication maybe from merging conflicts.

Reply:
It is not duplication. The types of `input` and `indices` are different from the existing definitions.

Reply:
Oh sorry I missed that part. Thanks!